client/state/action-types.js

```diff
@@ -52,6 +52,7 @@ export const JETPACK_CONNECT_CHECK_URL_RECEIVE = 'JETPACK_CONNECT_CHECK_URL_RECE
 export const JETPACK_CONNECT_DISMISS_URL_STATUS = 'JETPACK_CONNECT_DISMISS_URL_STATUS';
 export const JETPACK_CONNECT_AUTHORIZE = 'JETPACK_CONNECT_AUTHORIZE';
 export const JETPACK_CONNECT_AUTHORIZE_RECEIVE = 'JETPACK_CONNECT_AUTHORIZE_RECEIVE';
+export const JETPACK_CONNETCT_AUTHORIZE_RECEIVE_SITE_LIST = 'JETPACK_CONNETCT_AUTHORIZE_RECEIVE_SITE_LIST';
```

Comment:
`JETPACK_CONNETCT_AUTHORIZE_RECEIVE_SITE_LIST` -> isn't that `CONNETCT` a typo? :)